massa-node/src/settings.rs

```diff
@@ -386,6 +389,9 @@ pub struct VersioningSettings {
 pub struct BlockDumpSettings {
     /// Where to dump blocks
     pub(crate) block_dump_folder_path: PathBuf,
+    #[cfg(feature = "dump-block")]
```

Comment:
isn't the whole struct depending upon the feature "dump-block"?

Reply:
Yes we could do a cfg_attr on the whole structure and do it for all `Serialize` introduced in https://github.com/massalabs/massa/pull/4695 I open an issue